metrics/no_op_stats.go

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
```

Comment:
tests?

Reply:
Yeah will add test if the logic looks fine